gammapy/datasets/map.py

```diff
@@ -629,7 +631,9 @@ def create(
         )
 
         kwargs.update(geoms)
-        return cls.from_geoms(reference_time=reference_time, name=name, **kwargs)
+        return cls.from_geoms(
```

Comment:
If this was not there before, this probably means we miss a test to check that a `meta_table` is actually present if passed on creation.